src/main/webapp/WEB-INF/jsp/managestudy/jobs.jsp

```diff
@@ -85,6 +85,12 @@
   function formatDate(date) {
     return moment(date).format(dateFormat);
   }
+  function formatDateWithNull(date){
```

Comment:
update the formDate function to check for null instead of creating a new function

Reply:
Changes made as per the comments